Donation-platform/src/main/java/com/google/FindTrendingCharities.java

```diff
@@ -25,17 +25,18 @@
 import java.util.HashMap;
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.model.Charity;
+import com.google.model.Tag;
 
 public final class FindTrendingCharities {
 
-    //datastore set up
-    private DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
-    private DbCalls db = new DbCalls(ds);
+    DbCalls db;
 
     //number of trending charities to be returned
     final int trendingNum = 7;
```

Comment:
private static final int MAX_CHARITIES_TO_RETURN;
rename other constant definitions similarly. 
Use descriptive names that capture the usage of the variable

Reply:
done